pkg/addons/applier.go

```diff
@@ -62,10 +59,19 @@ func (a *Applier) Outro(ctx context.Context) error {
 	if err != nil {
 		return fmt.Errorf("unable to load addons: %w", err)
 	}
+
+	errCh := a.waitForKubernetes(ctx, kcli)
```

Comment:
how about running this in a defer statement?

Reply:
The 'read from errCh and print that message' bit?
yeah that makes sense to me